resources/i18n/nl.json

```diff
@@ -0,0 +1,49 @@
+{
+	"@metadata": {
+		"authors": [
+			"Dutchy45",
+			"Mainframe98"
+		]
+	},
+	"print": "Afdrukken",
+	"new-tab": "Nieuw tabblad",
+	"close-tab": "Tabblad sluiten",
+	"open-file": "Bestand openen",
+	"table-of-content": "Inhoudsopgave",
+	"zoom-in": "Inzoomen",
+	"zoom-out": "Uitzoomen",
+	"help": "Hulp",
+	"feedback": "Terugkoppeling",
+	"report-a-bug": "Probleem melden",
+	"exit": "Afsluiten",
+	"save-file-as-window-title": "Bestand opslaan als",
+	"file": "Bestand",
+	"edit": "Bewerken",
+	"view": "Beeld",
+	"search": "Zoeken",
+	"settings": "Instellingen",
+	"back": "Terug",
+	"forward": "Volgende",
+	"title": "Titel",
+	"size": "Grootte",
+	"date": "Datum",
+	"open": "Openen",
+	"delete": "Verwijderen",
+	"download": "Downloaden",
+	"resume": "Hervatten",
+	"pause": "Pauzeren",
+	"cancel": "Annuleren",
+	"apply": "Toepassen",
+	"about-version": "Versie : {{VERSIE}}",
```

Comment:
The `{{VERSION}}` token must not be translated.

Reply:
I've fix this with a specific commit. But it should be fixed "upstream".

Reply:
Imported the message into translatewiki.net

I've also added a validator for variables (strings in the format `{{VAR}}`) to ensure that they remain in translations as is.

![validator](https://user-images.githubusercontent.com/1685517/87548900-c6400800-c6ca-11ea-8ce0-14cd8cc06016.png)